src/pages/ContactsPage/container.jsx

```diff
@@ -122,7 +125,7 @@ class ContactsPage extends React.Component {
   };
 
   handleCreate = () => {
-    console.log("Create");
+    navigate('/contacts/create');
```

Comment:
I would do the logic of such buttons using `Link` component, but you can do it later.